### PR TITLE
Create Burwood InfoCouncil Scraper

### LINKED ISSUE #60: [New Scraper] NSW: Burwood
**Council name:**
Burwood Council

**Link to agenda webpage:**
https://www.burwood.nsw.gov.au/Our-Council/Council-and-Committee-Meetings/Minutes-and-Agendas

**Webpage quirks**
Nup, also accordion similar to Melbourne Council layouts.

**Additional context**
image

### PR DESCRIPTION
**Pull request by benfrstr**
Add a scraper for Burwood council. Closes #60.

### CODE REVIEW

File: scrapers/nsw/burwood.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import sys
+from pathlib import Path
+
+parent_dir = str(Path(__file__).resolve().parent.parent.parent)
+if parent_dir not in sys.path:
+    sys.path.append(parent_dir)
```
Comment (ArchangelEkim) [inline scrapers/nsw/burwood.py:6]: These lines are unneeded with the recent restructuring of the program.

File: scrapers/nsw/burwood.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+from base_scraper import register_scraper
+from infocouncil_scraper import InfoCouncilScraper
```
Comment (ArchangelEkim) [inline scrapers/nsw/burwood.py:9]: Update imports.

```{suggestion}
from council_scrapers.base import InfoCouncilScraper, register_scraper
```

**Review by ArchangelEkim — CHANGES_REQUESTED** [review]

**Review by benfrstr — COMMENTED** [review]
Thanks for the requested changes!

File: scrapers/nsw/burwood.py (hunk 2 above)
Comment (ArchangelEkim) [reply scrapers/nsw/burwood.py:9]: These changes need to be committed into the repo before I can approve

**Review by ArchangelEkim — CHANGES_REQUESTED** [review]

**Comment by benfrstr** [comment]
It's not letting me commit the changes, only submit feedback weirdly.

**Comment by ArchangelEkim** [comment]
I'll fix it from my end.

**Review by ArchangelEkim — APPROVED** [review]

**Comment by benfrstr** [comment]
Thank you!

**Pull request merged**